intake_esm/aggregate.py

```diff
@@ -188,13 +197,13 @@ def set_coords(ds, varname):
     return ds.set_coords(coord_vars)
 
 
-def open_dataset(url, data_vars, chunk_size=default_chunk_size):
+def _open_dataset(url, data_vars, chunk_size=config.get('default_chunk_size'), **kwargs):
```

Comment:
I think this may lead to misleading behavior. A user might expect modifying config to modify the defaults, but since the API is defined at import-time, this won't happen. I suggest `chunk_size=None` here and 
```python
if chunk_size is None:
  chunk_size = config.get('default_chunk_size')
```
in the function.

Reply:
I just realized that we may have a slight problem. One line 204, we are using `time` as the default time coordinate, and I am wondering whether there is any way to replace this assumption with something robust?

https://github.com/NCAR/intake-esm/blob/ef62f70610519f1197e5ed975d2f290d1440375d/intake_esm/aggregate.py#L201-L207

Reply:
We're expecting `chunks` in the args `_open_dataset`. Do we need to not chunk here because we're chunking later in `concat_ensembles` or do we want to subset `kwargs['chunks']` for the
`time_coord_name` entry?